Helper/Config.php

```diff
@@ -153,6 +154,17 @@ public function isStoreAlternativePaymentMethodEnabled($storeId = null)
         return $this->adyenHelper->getAdyenHppVaultConfigDataFlag('active', $storeId);
     }
 
+    /**
+     * Get 3DS1 flow behavior config
+     *
+     * @param null|int|string $storeId
+     * @return mixed
+     */
+    public function getThreeDsBehavior($storeId = null)
+    {
+        return $this->adyenHelper->getAdyenCcConfigData(self::XML_THREEDS_BEHAVIOR, $storeId);
```

Comment:
Please document it for Version 6 (also needs to be added to the release notes) and tag it that above version 7 it will be removed. In version 7 we handle all the redirects with the same logic.

Reply:
Docs have been updated, see ticket for more info.